e2e-tests/portal/cypress/e2e/pages/dashboardDeals.js

```diff
@@ -20,6 +20,10 @@ const page = {
   filters: {
     panel: {
       form: {
+        createdByYou: {
+          label: () => cy.get('[data-cy="filter-label-createdBy-Created-by-you"]'),
```

Comment:
the name is a bit repetitive - `...-createdBy-Created-by-you`. can we simplify it?

Reply:
changed - removed field